packages/core/src/app/ContextedApplication.tsx

```diff
@@ -109,7 +109,7 @@ const ContextedApplication = ({
       <Translator />
       <ErrorBoundary
         errorScreen={ErrorScreen}
-        userIdfetcher={getActiveUserId}
```

Comment:
Instead of modifying ErrorBoundary object, I think it could be a better practice to define a callback function which return the same format as before using getActiveUserInfo

`
const getActiveUserId = useCallback(() => getActiveUserInfo().then(({userId}) => userId), [...]);
`